emulfile.py

```diff
@@ -29,9 +29,9 @@
 # Import all the exceptions
 from exception_hierarchy import *
 
-# Fix for ticket #983. By retaining a reference to unicode, we prevent
-# os.path.abspath from failing in some versions of python when the unicode
-# builtin is overwritten.
+# Fix for SeattleTestbed/ATTIC#983.
```

Comment:
I'd make `ATTIC` lower case, like GitHub's markdown would be able to resolve it:

SeattleTestbed/attic#983 vs. SeattleTestbed/ATTIC#983

Reply:
Whoa, good catch! We definitely want the link to work.